articles/libraries/auth0-swift/index.md

```diff
@@ -137,8 +137,10 @@ Auth0
     }
 ```
 
+If you're using **Swift 5+** the `Auth0.Result` type may shadow Swift's `Result`. To prevent that, whenever you want to refer to Swift's type, use `Swift.Result` instead. This will be fixed in the next major version of Auth0.swift.
```

Comment:
```suggestion
If you're using **Swift 5+**, `Auth0.Result` may shadow Swift's `Result` type. To prevent that, replace it with `Swift.Result` whenever you want to refer to Swift's built-in type. This will be fixed in the next major version of Auth0.swift.
```

Reply:
Thanks @stevehobbsdev!